style.css

```diff
@@ -0,0 +1,113 @@
+body {
+  background-color: #f6eada;
+  color:#128057
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Jost", sans-serif;
+}
+
+.filter-bar {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 50px;
+}
+
+header {
+  text-align: center;
+  font-size: 2rem;
+  font-weight: 500;
+  background-color: #ee4574;
+}
+
+#title {
+  margin-top: 2rem;
+}
+
+.gallery-wrapper {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  column-gap: 35px;
+  row-gap: 20px;
+  margin: 40px 40px 40px 40px;
+} 
+
+.card-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  max-width: 100%;
+  padding: 16px, 16px, 24px, 16px;
+  box-sizing: border-box;
+  border-radius: 20px;
+  border: dotted #ee4574;
+  cursor: pointer; 
+}
+
+.card-wrapper:hover {
```

Comment:
When you apply an effect to the card-wrapper, like in this case the change in size, is making all the UI around move, and this is to be avoided. A solution would be to wrap the element in a div and give the div a fixed height 👍 Or try a less impactful UI change, like just change colours?